storybook/stories/API/props/ChartProps.ts

```diff
@@ -45,6 +45,16 @@ export const margin = {
   },
 };
 
+export const dataKey = {
+  description: 'The key or getter of a gbroup of data which should be unique in a LineChart',
```

Comment:
typo: `gbroup` -> `group`

Also, where is this used? It looks like right now its specific to a `LineChart`?

Reply:
> typo: `gbroup` -> `group`
> 
> Also, where is this used? It looks like right now its specific to a `LineChart`?

@ckifer This is showing in [the Funnel API docs](https://recharts.org/en-US/api/Funnel). Is this not right? Should it be removed?

Reply:
It definitely shouldn't be `LineChart`, so I think a typo in the old docs. I would say replace with `FunnelChart`

if you look at [`Line` ](https://recharts.org/en-US/api/Line)its identical, so I think a copy-pasta

Reply:
@ckifer Fixed!